web-ui/src/pages/UploadNotesPage.jsx

```diff
@@ -19,7 +19,7 @@ const HomePage = () => (
         <p>
             If you should require a copy of any of the documents that you
             have submitted here, please email
-            <a href="mailto:hr@objectcomputing.com">hr@objectcomputing.com</a>
+            <a href="mailto:hr@objectcomputing.com"> hr@objectcomputing.com</a>
```

Comment:
wha?

Reply:
Oops I meant to fix the space issue there, I'm about to push to fix it.